extensions/sql-migration/src/constants/strings.ts

```diff
@@ -82,8 +82,50 @@ export const XEVENTS_ASSESSMENT_DESCRIPTION = localize('sql.migration.database.a
 export const XEVENTS_ASSESSMENT_HELPLINK = localize('sql.migration.database.assessment.xevents.link', "Learn more");
 export const XEVENTS_ASSESSMENT_OPEN_FOLDER = localize('sql.migration.database.assessment.xevents.instructions', "Select a folder where extended events session files (.xel and .xem) are stored");
 
+// Assessment results
+export const ASSESSMENT_RESULTS_PAGE_TITLE = localize('sql.migration.assessment.results.title', "Target Platform & Assessment Results");
+export const ASSESSMENT_RESULTS_SUMMARY_LABEL_CAPS = localize('sql.migration.assessment.results.summary.label.caps', "ASSESSMENT RESULTS");
+export const ASSESSMENT_RESULTS_PAGE_HEADER = localize('sql.migration.assessment.results.header', "Choose Target platform, view assessment results and select database(s) for migration.");
```

Comment:
"Target" should not be capitalized.  "Choose target..."

Reply:
done